lib/build.js

```diff
@@ -46,7 +46,7 @@ function build (gyp, argv, callback) {
         }
         return
       }
-      config = JSON.parse(data.replace(/#.+\n/, ''))
+      config = JSON.parse(data.replace(/#.+[\n\r]+/, ''))
```

Comment:
the `[]` is making this do more than is intended, let's be minimal and just turn `\n` into `\r?\n` and that should solve the Windows problem

Reply:
```suggestion
      config = JSON.parse(data.replace(/#.+\r?\n/, ''))
```